api/envoy/extensions/filters/http/oauth2/v3alpha/oauth.proto

```diff
@@ -74,6 +74,11 @@ message OAuth2Config {
 
   // Any request that matches any of the provided matchers will be passed through without OAuth validation.
   repeated config.route.v3.HeaderMatcher pass_through_matcher = 8;
+
+  // Optional list of OAuth scopes to be claimed in the authorization request. If not specified,
+  // defaults to "user" scope.
```

Comment:
Was this behavior present before (defaulting to "user")? Just checking in for API backwards compat. That said, this is v3alpha, so I think we can allow the change anyway.

Reply:
@htuch Yes, sure. Actually this issue raised up because the "user" value was hardcoded in the query param string. The objective here is to make this really configurable. So using the default value of "user" will make it backward compatible.